README.md

```diff
@@ -625,7 +644,7 @@ He can run the role with the following variables to install Nextcloud accordingl
      nextcloud_mysql_root_pwd: "42h2g2"
      nextcloud_apps:
        files_external: "" # enable files_external which is already installed in nextcloud
-       calendar: "https://github.com/nextcloud/calendar/releases/download/v1.5.0/calendar.tar.gz"
+       calendar:  # Download from official store
```

Comment:
Could you define some test for molecule by adding test scenario (the best would be app from zip file and app from official store) - similar to this (add whatever app you want which is not installed by default):
```
  vars:
    nextcloud_apps:
      calendar:
      forms: https://github.com/nextcloud/forms/releases/download/v3.0.3/forms.tar.gz
```
here: https://github.com/aalaesar/install_nextcloud/blob/c9d4510edaf45ac8700e8dee21e52c777907b1e8/molecule/default/latest-converge.yml#L12-L13

Reply:
I have no idea about molecule. I used this:
     nextcloud_apps:
       calendar:
       contacts:
         source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.0.2/contacts-v5.0.2.tar.>
         conf:
           var_1: "no"
       passwords:

Reply:
nextcloud_apps:
       calendar:
       contacts:
         source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.0.2/contacts-v5.0.2.tar.>
         conf:
           var_1: "no"
       passwords:

Reply:
> ```
>  nextcloud_apps:
>    calendar:
>    contacts:
>      source: "https://github.com/nextcloud-releases/contacts/releases/download/v5.0.2/contacts-v5.0.2.tar.>
>      conf:
>        var_1: "no"
>    passwords:
> ```

that's a very nice test case.
Can you add it to [install_nextcloud/molecule/default/latest-converge.yml](https://github.com/aalaesar/install_nextcloud/blob/c9d4510edaf45ac8700e8dee21e52c777907b1e8/molecule/default/latest-converge.yml)  ?

Reply:
It's good test scenario. :) Add it in line 13 as I mentioned above within: `vars:` block.

Reply:
OK, included @line13 in a vars-block. Thanks for this detailed explanations and I hope I got it right.